targets/xorg

```diff
@@ -108,6 +108,8 @@ install xorg arch=,xserver-xorg-video-fbdev$ltspackages
 if [ "${ARCH#arm}" = "$ARCH" ]; then
     install arch=,xserver-xorg-video-intel$ltspackages
 fi
+# JMT: eventually wrap this removal with an appropriate test
+remove xf86-video-odroid-c1
```

Comment:
That will still break on non-arm, non-arch: xf86-video-odroid-c1 does not exist on non-arm arch.

So you should put it in the else branch of the if statement above, and you probably need to change the line to `remove arch=xf86-video-odroid-c1,`. But, again, I'd rather change the install line and only install necessary drivers.